src/iocore/hostdb/HostDBInfo.cc

```diff
@@ -65,6 +65,17 @@ HostDBInfo::assign(IpAddr const &addr) -> self_type &
   return *this;
 }
 
+/** Assign SRV record and name
+ *
+ * @param srv Pointer to SRV to assign.
+ * @param name Pointer to hostname for the record
```

Comment:
nit: Can you please make these both `@param[in]` to show they are input variables. Along these lines, I suggest: `Pointer to the SRV from which to assign.`

Reply:
Thank you!